__tests__/screens/send-bitcoin-screen.tsx

```diff
@@ -0,0 +1,352 @@
+import * as React from "react"
+import { MockedProvider } from "@apollo/client/testing"
+import { gql, InMemoryCache } from "@apollo/client"
+import { act, cleanup, fireEvent, render, waitFor } from "@testing-library/react-native"
+import moment from "moment"
+import "@testing-library/jest-native/extend-expect"
+import "../../node_modules/react-native-gesture-handler/jestSetup.js"
+
+import { translate } from "../../app/i18n/translate"
+import "../../__mocks__/react-native-firebase"
+import {
+  LIGHTNING_PAY,
+  PAY_KEYSEND_USERNAME,
+  SendBitcoinScreen,
+} from "../../app/screens/send-bitcoin-screen"
+import { QUERY_PRICE, WALLET } from "../../app/graphql/query"
+
+jest.mock("@react-navigation/native", () => {
```

Comment:
is this a common mock to multiple test suites? if so, please add it in a setup or in its own mock file